dockstore-webservice/src/main/resources/import.sql

```diff
@@ -4,6 +4,21 @@
 -- column remark support in JPA does not seem to work for postgres, possible starting point https://stackoverflow.com/questions/28773022/jpa-column-annotation-to-create-comment-description 
 COMMENT ON COLUMN sourcefile_verified.platformversion IS 'By default set to null';
 COMMENT ON COLUMN validation.message IS 'A mapping of file path to message.';
+COMMENT ON COLUMN tool.lastbuild IS 'For automated builds: When refresh is hit, the last time the tool was built gets stored here. If tool was never built on quay.io, then last build will be null. N/A for hosted/manual path tools';
+COMMENT ON COLUMN tool.lastupdated IS 'For automated builds: last time tool/namespace was refreshed Dockstore, tool info updated, default version selected. For hosted tools: when you created the tool';
+COMMENT ON COLUMN tool.lastmodified IS 'For automated builds: N/A. For hosted: Last time a file was updated (new version created)';
+COMMENT ON COLUMN tool.dbcreatedate IS 'For automated builds and hosted: Time registered on Dockstore, either by refresh or manual register. Can be blank as this column was added in 2018.';
+COMMENT ON COLUMN tool.dbupdatedate IS 'For automated builds: Last time tool/namespace was refreshed, different version is selected, checker workflow was added, or tool info updated (like path information). For hosted: Last time a file was updated (new version created), default version selected. Can be blank as this column was added in 2018. Basically anytime db entry modified';
```

Comment:
I think it should be `any time` instead of `anytime`, but I'm not convinced that is so. :)